torch/nn/quantizable/modules/rnn.py

```diff
@@ -0,0 +1,294 @@
+import numbers
+from typing import Optional, Tuple
+import warnings
+
+import torch
+from torch import Tensor
+
+"""
+We will recreate all the RNN modules as we require the modules to be decomposed
+into its building blocks to be able to observe.
+"""
+
+class LSTMCell(torch.nn.Module):
+    _FLOAT_MODULE = torch.nn.LSTMCell
+
+    def __init__(self, input_dim, hidden_dim, bias=True):
```

Comment:
can we make sure mypy is enabled for this module (in caffe2/mypy.ini), and annotate types on at least function I/O?   It would help with readability.

Reply:
SG!